docx/header.py

```diff
@@ -17,3 +17,23 @@ class Header(ElementProxy):
     """
 
     __slots__ = ()
+    type = 'default'  # just hardcoded for now
+
+    @property
+    def is_linked_to_previous(self):
+        """
+        Boolean representing whether this Header is inherited from
+        a previous section.
+        """
+        # self._element should probably be renamed self._sectPr,
+        # like it is on Section
+        if self._element.has_header_reference(self.type):
+            return False
+        else:
+            return True
```

Comment:
* We'll be needing an enumeration here I think, the WdHeaderFooterIndex one. If we do need type, we'll want to store it as an instance variable because we should be able to reuse this class for any header. It will need it's own __init__() then of course that calls it's parent's __init__() with the needful; and __slots__ will need a place for the instance variable.
* We'll want to stay as general purpose as possible, so I'm thinking a method get_headerReference_of_type(type). This will come in handy later as well. It should return the matching headerReference element or None if not present.